scripts/eslint-plugin/rules/hook-super-invocation.js

```diff
@@ -0,0 +1,81 @@
+"use strict";
+
+const JSDOM_HOOK = new Set([
+  "_attach",
+  "_detach",
+  "_attrModified",
+  "_descendantAdded",
+  "_descendantRemoved",
+  "_childTextContentChangeSteps"
+]);
```

Comment:
It might be more consistent with the other rules to define these method names as [options](https://eslint.org/docs/developer-guide/working-with-rules#contextoptions) in the ESLint config file rather than hardcoding them here.

Reply:
I agree it would be nicer if we could configure linting-related things in one place. (I.e., if/when we add a future hook.) If it's not too much work that'd be preferable, but if it is, then feel free to leave them here.